tags/reserved_tags.go

```diff
@@ -147,6 +147,19 @@ const (
 	XAkitaDumpFilterFlag Key = "x-akita-dump-filter-flag"
 )
 
+// Tags applied to a copy of the spec
+const (
+	XAkitaOriginalOrganization = "x-akita-original-organization"
```

Comment:
Is this an ID or a name?

Reply:
Changed to use ID suffix to be consistent.